Configuration/PyReleaseValidation/python/relval_nano.py

```diff
@@ -155,6 +155,28 @@ def subnext(self):
                              '--conditions':'auto:phase1_2022_realistic'},
                             _NANO_mc])
 
+##13.X INPUT
+steps['RunScoutingPFRun32022D13.X']={'INPUT':InputInfo(dataSet='/ScoutingPFRun3/Run2022D-v1/RAW',label='2022D',events=100000,location='STD', ls=Run2022D)}
+
+steps['NANO_dataRun3ScoutingPF13.X']=merge([{'-s':'NANO:PhysicsTools/NanoAOD/custom_run3scouting_cff',
+                         '--conditions':'auto:run3_data',
+                         '-n':'10',
+                         '--era' : 'Run3',
+                         '--geometry' : 'DB:Extended',
+                         '--datatier':'NANOAOD',
+                         '--eventcontent':'NANOAOD'}])
+
+steps['NANO_mcRun3ScoutingPF13.X']=merge([{'-s':'NANO:PhysicsTools/NanoAOD/custom_run3scouting_cff.nanoSequenceMC',
+                         '--conditions':'auto:phase1_2022_realistic',
```

Comment:
I believe that the `.nanoSequenceMC` is not necessary, since `--mc` will have cmsDriver pick that one up automatically. We'll deal with this at a later stage during the `NANO:@Scouting` integration